x-pack/plugins/cases/public/types.ts

```diff
@@ -51,7 +51,19 @@ export type StartServices = CoreStart &
     security: SecurityPluginSetup;
   };
 
+type CaseOwnerTypes = 'securitySolutionCases' | 'observabilityCases';
```

Comment:
`type CaseOwnerTypes = SECURITY_SOLUTION_OWNER | OBSERVABILITY_OWNER;`

Reply:
its a little different, `'securitySolutionCases' | 'observabiltyCases'` is the name of the capabilities, but the owner is called `'securitySolution' | 'observability'`

Reply:
@semd , Actually, are you suggesting it would be better if externally it was still referred to as securitySolution, as we take care of slapping the Cases on the end of it: capabilities[`${owner}Cases`]